.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,39 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
 -->
+
+#### What this PR does / why we need it:
+
+<!---
+Please put your overall PR description here
+-->
+
+#### How to verify it
+
+<!---
+Please specify the precise conditions and/or the specific test(s) which must pass.
+-->
+
+#### Which issue(s) this PR fixes:
+
+<!--
+Please uncomment this block and include only one of the following on a
+line by itself:
+
+None
+
+-OR-
+
+Fixes #<issue number>
+
+*** Please also put 'Fixes #' in the PR title and commit description***
```

Comment:
Fixes should not be part of the PR title

Reply:
@mheon wanted this.

Reply:
It should be part of the PR description but not the title.

Reply:
I'm not opposed to whatever, I'm just pretty sure Matt wanted it in the title and description, but maybe I'm wrong. His comment is long since lost to `--force` push history :confused:

Reply:
Description is better than title, sorry if I was unclear.

Reply:
@mheon when you have a moment, could you clarify what/where you'd like the `FIXES #<whatever>` text to appear?

Reply:
PR description and commit description both. Can be in the title, but does not need to be.

Reply:
Got it, thanks for clarification.